copier.yml

```diff
@@ -214,8 +214,12 @@ gitlab_url:
 domains_prod:
   type: yaml
   multiline: true
-  default: {}
+  default: null
   placeholder: |
+    - hosts:
```

Comment:
I don't see the need of this, as there's already the same 3 lines below. Maybe we should add `cert_resolver: letsencypt` by default. cc @josep-tecnativa

Reply:
I have removed duplicate lines.
The placeholder has the comment in the end, to show first the syntax of the value.